sklearnex/preview/covariance/covariance.py

```diff
@@ -100,4 +106,27 @@ def fit(self, X, y=None):
 
         return self
 
+    # This needs to be included to guarantee that we use the sklearnex pairwise_distances
```

Comment:
why is this necessary? we don't support this metric in sklearnex anyway, so, the default one still will be called.